### PR TITLE
Ci/update gha workflows

### PR DESCRIPTION
**Pull request by PeterC89**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Updates GHA workflows to remove depreciated calls

* **What is the current behavior?** (You can also link to an open issue here)
Workflows run successfully

* **What is the new behavior (if this is a feature change)?**
Workflows continue to run successfully with the same behaviour

* **Other information**:

**Status**

- [ ] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [ ] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: .github/workflows/node.yaml [hunk 1]
```diff
@@ -175,17 +174,16 @@ jobs:
...
       - name: Determine images to publish
         id: image-tags
         # TODO - image needs changing...
         run: |
           IMAGES=
-          DOCKER_TAG=${{ steps.docker-tag.outputs.tag }}
+          if [[ ${{ github.ref_name }} == "master" || ${{ github.ref_name }} == "main" ]]
+            DOCKER_TAG=latest
+          else
+            DOCKER_TAG=${{ github.ref_name }}
```
Comment (baltedewit) [inline .github/workflows/node.yaml:185]: Are special characters allowed in the docker tag or should it have the same filter applied as elsewhere?

**Review by baltedewit — APPROVED** [review]

**Comment by Julusian** [comment]
Silje has already been reworking the workflows in this repository, so I don't think there will be anything left to do here
https://github.com/nrkno/sofie-core/pull/803 https://github.com/nrkno/sofie-core/pull/811

File: .github/workflows/node.yaml (hunk 1 above)
Comment (PeterC89) [reply .github/workflows/node.yaml:185]: It appears they are not, but it didn't have it before so I didn't add it, probably should!
The docker website says:
`A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.`

I wonder if we should add a check for length as well then...

**Review by siljekristensen — APPROVED** [review]

**Comment by siljekristensen** [comment]
> > Silje has already been reworking the workflows in this repository, so I don't think there will be anything left to do here #803 #811
> 
> Ah whoops, I thought all that work had been merged, I'll check!

I hadn't touched this workflow, so these changes are needed :)

**Review by jstarpl — APPROVED** [review]

**Pull request merged**